conans/test/integration/conanfile/required_conan_version_test.py

```diff
@@ -77,3 +77,17 @@ class Lib(ConanFile):
                             """)
         client.save({"conanfile.py": conanfile})
         client.run("export . --name=pkg --version=1.0")
+
+    def test_commented_out_required_conan_version(self):
+        """Used to not be able to comment out required_conan_version"""
+        client = TestClient()
+        conanfile = textwrap.dedent("""
+                            from conan import ConanFile
+                            # required_conan_version = ">=100.0"
```

Comment:
I am not sure how this failed before. Probably the ``import`` need to fail too? and ``assert_error=True``, but version not validated?

Reply:
You were right, thanks for the headsup!